src/exceptions/httpException.ts

```diff
@@ -0,0 +1,13 @@
+class HttpException extends Error {
```

Comment:
I would purpose more gently work with errors, like not extend them from `Error` to got Error `stack`, by this they will be performant. 

Just like idea to discuss or implement, we have **exceptions** which we know: where and why they appears, so `stack` here is redundant, if you create meaningful message; and we have surprisingly coming **errors** -- where we want **all** info for investigation.

Reply:
Yes, can discuss it on our workshop. For now it is deleted.